docs/mindsdb-docs/docs/setup/custom-config.md

```diff
@@ -0,0 +1,75 @@
+# Custom Configuration of MindsDB Locally
+
+To follow this guide, please make sure you have a local installation of MindsDB. [Here](https://docs.mindsdb.com/setup/self-hosted/pip/source/), you can find out how to install MindsDB locally.
+
+## Starting MindsDB with Default Configuration
+
+It is very straightforward to start MindsDB locally with the default config file - just run the commands below.
+
+```bash
```

Comment:
Let's mention that users need to activate virtual env with this command

Reply:
Done.